api/python/quilt3/main.py

```diff
@@ -207,16 +208,30 @@ def _selector_fn_no_copy(*args):
     return False
 
 
-def cmd_push(name, dir, registry, dest, message, meta, workflow, force, dedupe, no_copy):
+def cmd_push(name, dir, registry, dest, message, meta, workflow, force, dedupe, no_copy, request, dir_logical_key):
     if util.PhysicalKey.from_url(util.fix_url(dir)).is_local() and no_copy:
```

Comment:
I'm not convinced "request" and "dir_logical_key" are the right names.  Anyone have better suggestions?